201-vm-domain-join/azuredeploy.json

```diff
@@ -14,15 +14,15 @@
         "description": "Existing subnet that contains the domain controller"
       }
     },
-    "dnsLabelPrefix": {
+    "vmName": {
```

Comment:
Let's keep this `dnsLabelPrefix` since it sets the right expectations for the uniqueness and the regex. Otherwise it causes a lot of failures due to it not being unique.